### DISCUSSION TITLE
[Question] Tooltip position관련 문의드립니다.

### DISCUSSION THREAD

**Discussion by yangSangHoon** [wanteddev/montage-web#13] [Q&A]
### 요약

<TooltipContent> 의 위치를 변경하고 싶은데 방법이 있을까요?

### 상세

아래 이미지 보시면 툴팁을 조금 더 위로 올리고 싶어서 문의 드렸습니다 :)
![Screenshot 2024-05-22 at 4 26 02 PM](https://github.com/wanteddev/wds/assets/15603335/3c33a46c-1428-48d9-a0c9-84292bd6baa3)

**Comment by Sh031224** [comment]
툴팁은 현재 offset이 고정되어 있습니다..!
offset을 수정할 수 있도록 개선해보겠습니다
